recipes/whatsapp/webview.js

```diff
@@ -48,6 +87,7 @@ module.exports = Ferdium => {
   const loopFunc = () => {
     getMessages();
     getActiveDialogTitle();
+    addThemeMessage();
```

Comment:
Will this keep popping up every second and be a bad UX?

Reply:
The function will trigger every second (or every time frame we have it implemented). But the function will only make changes as per this line (22) `if (!elementExists)`